### DISCUSSION TITLE
[Feature Request] Open Ended Year Filters

### DISCUSSION THREAD

**Discussion by RadiantRust** [autobrr/autobrr#1797] [Ideas]
In a filter, the year field currently accepts comma separated year values and ranges where both ends are defined. 

Could this be expanded to allow for open ended ranges? 

- *-2000 - Any Year before 2000
- 2020-* Any year after 2020

**Comment by zze0s** [comment]
That is over complicating things - just do 1900-2000 and 2020-2100. Maybe it will be implemented before 2100 :)

**Reply by RadiantRust** [reply]
Fair enough, I can see how it might cause problems! I don't know why setting years in the future never occured to me. I'll do that, Thanks!
